_data/simple-icons.json

```
@@ -10985,6 +10985,11 @@
             "hex": "39457E",
             "source": "https://github.com/tpf/marketing-materials/blob/6765c6fd71bc5b123d6c1a77b86e08cdd6376078/images/onion-logo/tpf-logo-onion.svg"
         },
+        {
+            "title": "perplexity",
```

```suggestion
            "title": "Perplexity",
```

This also needs to be uppercase.